include/picongpu/param/radiationObserver.param

```diff
@@ -35,6 +35,9 @@ namespace picongpu
         {
             namespace radiation_observer
             {
+                /* float type used in radiation direction calculations */
+                typedef picongpu::float_X floatRadObserver;
```

Comment:
please use `using floatRadObserver = picongpu::float_X`

We do not use `typedef` anymore.

Reply:
Can we shorten this name? The declaration is already in the namespace `radiation_observer` therefore radiation and observer should not be part of the name, or shortened.

I suggest `float_ob`, `float_obs` or `float_dir`

Reply:
Will do this.

Reply:
I am not sure why is this type needed as public API at all? It seems just a details of that function below.

Reply:
It is not needed as public API, but should be user defined - what would you recommend instead @sbastrakov ?

Reply:
I think this type can be moved inside the function. With a comment to a user that it can be changed just here to control precision vs. calculation time